src/main/java/com/uid2/shared/vertx/RequestCapturingHandler.java

```diff
@@ -32,11 +34,21 @@ public class RequestCapturingHandler implements Handler<RoutingContext> {
     private Queue<String> _capturedRequests = null;
     private final Map<String, Counter> _apiMetricCounters = new HashMap<>();
     private final Map<String, Counter> _clientAppVersionCounters = new HashMap<>();
+    private ISiteStore siteStore;
 
     private static String formatRFC1123DateTime(long time) {
         return DateTimeFormatter.RFC_1123_DATE_TIME.format(Instant.ofEpochMilli(time).atZone(ZONE_GMT));
     }
 
+    public RequestCapturingHandler()
```

Comment:
Is this constructor still needed?

Reply:
I checked it is needed
> Java does not provide a default no-argument constructor if you define any other constructor.

Reply:
It is just a super cautious step for backward compatible - in case someone with old operator wants to use the new shared lib.

Reply:
At some point after all uses are replace we should get rid of this. Should it be marked as deprecated?